main.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+
+	"github.com/Clever/fluentd-kinesis-forwarder-monitor/config"
+)
+
+var log = logger.New("fluentd-kinesis-forwarder-monitor")
+var sfxSink = sfxclient.NewHTTPSink()
+
+type posInfo struct {
+	logFile string
+	inode   uint64
+	offset  int64
+}
+
+func sendToSignalFX(timestamp time.Time) error {
+	points := []*datapoint.Datapoint{}
+	dimensions := map[string]string{
+		"hostname": config.HOSTNAME,
+		"scope":    config.ENV_SCOPE,
+	}
+
+	datum := sfxclient.Gauge("fluentd-kinesis-forwarder-monitor", dimensions, timestamp.Unix())
+	points = append(points, datum)
+
+	return sfxSink.AddDatapoints(context.Background(), points)
+}
+
+func main() {
+	config.Initialize()
+
+	sfxSink.AuthToken = config.SIGNALFX_API_KEY
+
+	for {
+		ts, context, err := trackTimestamp(config.LOG_FILE_POS)
+		if err != nil {
+			log.ErrorD("track-timestamp", logger.M{"msg": err.Error()})
+		} else {
+			log.GaugeIntD("track-timestamp", int(ts.UnixNano()), logger.M{
```

Comment:
That's true.  I forget that this container is gonna running on a 100+ machines.  I'll up the timeout to 15 seconds, log once a minute.